Bio/SeqIO/IgIO.py

```diff
@@ -37,8 +37,8 @@ def __init__(self, source):
 
         Examples
         --------
-        >>> with open("IntelliGenetics/TAT_mase_nuc.txt") as handle:
-        ...     for record in IgIterator(handle):
+        >>> with open("IntelliGenetics/TAT_mase_nuc.txt") as stream:
+        ...     for record in IgIterator(stream):
         ...         print("%s length %i" % (record.id, len(record)))
```

Comment:
Maybe change the following to an f-string too?

``print(f"{record.id} length {len(record)}")``